test/endpoints/tchannel/baz/baz_simpleservice_method_call_test.go

```diff
@@ -178,9 +179,10 @@ func TestCallTChannelSuccessfulRequestOKResponse(t *testing.T) {
 		"endpointThriftMethod": "SimpleService::Call",
 		"endpointHandler":      "call",
 		"x-uuid":               "uuid",
-		"deviceversion":        "1.0",
-		"device":               "ios",
-		"regionname":           "sf",
+		"Deviceversion":        "1.0",
+		"Device":               "ios",
+		"Regionname":           "sf",
+		"x-token":              "token",
```

Comment:
The change in this PR should have no effects on the currently log fields except filtering.

Reply:
Done, removed x-token and the other changes are for the casing.